entity-view/impl/src/main/java/com/blazebit/persistence/view/impl/metamodel/AbstractAttribute.java

```diff
@@ -392,7 +392,7 @@ public static String replaceThisFromMapping(String mapping, String root) {
             return Collections.emptyMap();
         }
 
-        context.getTypeValidationExpressionFactory().createSimpleOrObjectExpression(expression, false).accept(visitor);
+        context.getTypeValidationExpressionFactory().createSimpleExpression(expression, false).accept(visitor);
```

Comment:
@beikov Why has `createSimpleOrObjectExpression` been removed at this point?

Reply:
There is a new form for this now `public Expression createSimpleExpression(String expression, boolean allowOuter, boolean allowQuantifiedPredicates, boolean allowObjectExpression)`